content/src/denylist/Denylist.ts

```diff
@@ -1,141 +1,41 @@
 import { Timestamp } from 'dcl-catalyst-commons'
-import { AuthChain, EthAddress } from 'dcl-crypto'
-import { validateSignature } from 'decentraland-katalyst-commons/signatures'
+import { AuthChain } from 'dcl-crypto'
 import { DenylistRepository } from '../repository/extensions/DenylistRepository'
-import { Repository } from '../repository/Repository'
-import { DB_REQUEST_PRIORITY } from '../repository/RepositoryQueue'
-import { ContentAuthenticator } from '../service/auth/Authenticator'
-import { ContentCluster } from '../service/synchronization/ContentCluster'
 import { DenylistTarget, DenylistTargetId, DenylistTargetType } from './DenylistTarget'
 
-export class Denylist {
-  constructor(
-    private readonly repository: Repository,
-    private readonly authenticator: ContentAuthenticator,
-    private readonly cluster: ContentCluster,
-    private readonly network: string
-  ) {}
+export abstract class Denylist {
```

Comment:
Just curious, any reason why this is an abstract class and not an interface?

Reply:
I used an abstract class as I left the static implementations inside it, does it make sense?